id.css

```diff
@@ -0,0 +1,78 @@
+/** 
+  * id.css
```

Comment:
shouldn't organize css files by ids vs classes imo, but by functionality ie. all main layout css in one file, all modal css in one file. likely in alphabetical order? not sure how id vs class css is ordered, @mattfung99 ?